package.json

```diff
@@ -13,19 +13,34 @@
   },
   "scripts": {
     "test": "jest",
-    "test-debug": "node --debug-brk --inspect ./node_modules/.bin/jest -i"
+    "test:watch": "jest --watch",
+    "test:debug": "node --debug-brk --inspect ./node_modules/.bin/jest -i",
+    "test:karma": "karma start --single-run",
+    "test:travis": "npm run lint && npm run test",
+    "lint": "node_modules/.bin/eslint .",
+    "style": "npm run lint"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/spanishdict/equivalency.git"
   },
   "devDependencies": {
-    "debug": "4.0.1",
+    "chai": "4.1.2",
     "eslint": "5.6.0",
     "eslint-config-prettier": "3.0.1",
-    "eslint-plugin-jest": "21.22.0",
+    "eslint-plugin-jest": "21.22.1",
     "eslint-plugin-prettier": "2.6.2",
+    "expect": "23.6.0",
```

Comment:
Expect is now part of Jest. So, Jest runs its own test suite, but when tests are run using Karma (as on Browserstack), Mocha is the test runner using Jest expect. (Jest doesn't run in the browser.)

Reply:
Thanks, good note. Should we remove `jest`?

Reply:
It's faster than the BS tests, and I think there are some limits on our BS account which conceivably we might run up against, so I lean towards defaulting to jest for development, and running cross-browser tests for PRs (or manually as needed).

Reply:
That also WFM. I guess I was suggesting "remove `jest`" === running `karma` just with headless chrome for development.